src/lib.rs

```diff
@@ -55,8 +55,11 @@ pub mod _benchable {
 /// >                ; any VCHAR, except delimiters
 /// > ```
 #[inline]
-fn is_token(b: u8) -> bool {
-    b > 0x1F && b < 0x7F
+fn is_method_token(b: u8) -> bool {
+    match b {
+        b'A'..=b'Z' => true,
```

Comment:
So that I'm sure I understood what you were saying, this range match is inlined here because it's faster _if the match is true_ than reaching into the token map? (That'd make sense, I assume this is compiles to a couple cmps, vs having to load from the map that's perhaps not in cache.)

If so, would you mind adding a little comment here that it's inline for perf reason? So Future Us doesn't go "what's this, debleeted". :)

Reply:
Certainly!